component-event-stream/cfn.yaml

```diff
@@ -93,8 +93,7 @@ Resources:
                 requestParameters:
                   "integration.request.header.Content-Type": "'application/x-amz-json-1.1'"
                 requestTemplates:
-                  application/json: !Sub "{\n    \"StreamName\": \"support-epic-view-event-stream-${Stage}\",\n    \"Records\": [\n{ \"Data\": \"$util.base64Encode($input.json('$'))\", \"PartitionKey\": \"1\"\n }\n]\n}"
-                  application/x-amz-json-1.1: !Sub "{\n    \"StreamName\": \"support-epic-view-event-stream-${Stage}\",\n    \"Records\": [\n{ \"Data\": \"$util.base64Encode($input.json('$'))\", \"PartitionKey\": \"1\"\n }\n]\n}"
+                  application/json: !Sub "#set($payload = \"$input.json('$')\n\") \n {\n    \"StreamName\": \"support-epic-view-event-stream-${Stage}\",\n    \"Records\": [\n{ \"Data\": \"$util.base64Encode($payload)\", \"PartitionKey\": \"1\"\n }\n]\n}"
```

Comment:
Does AWS process this Velocity? template? Or is there some preprocessor in the build script which does it?

Reply:
yes this is run by api gateway.
The change here is to add a newline at the end of the serialized event, because otherwise firehose just writes it all to a file in s3 without newlines